extension-style/src/main/java/com/mapbox/maps/extension/style/localization/Localization.kt

```diff
@@ -0,0 +1,88 @@
+package com.mapbox.maps.extension.style.localization
```

Comment:
should it be part of the style extension?

Reply:
It's functionally independent, so not related to style extension and